.babelrc

```diff
@@ -1,6 +1,17 @@
 {
   "presets": [
-    "es2015",
+    ["env", {
+      "targets": {
+        "browsers": [
+          "last 2 versions",
+          "not ie < 13",
+          "not android < 50"
```

Comment:
The Discovery app is meant to be Desktop only

Reply:
Browserlist does not offer any option to do desktop only. I already excluded old android browser, so the babel plugins are on a minimal already.

I am pretty sure it is fine like that :)